index.html

```diff
@@ -79,6 +100,13 @@ <h4> Snack </h4>
           </article>
           <article class="snacks">
           </article>
+          <article class="total-calories">
+            <p class="total-calorie">
```

Comment:
I think we can maybe remove the 'p' tags for each of these.. I'm not sure how yet, but it feels like we're just nesting each element another layer deep. Let's revisit this later!